docker-compose/core-cc/config/data-bridge/core-cc-gridcapa-cbcora-data-bridge-application.yml

```diff
@@ -2,10 +2,10 @@ data-bridge:
   zone-id: "Europe/Paris"
   target-process: CORE_CC
   file-type: CBCORA
-  file-regex: .*CORESO-F301_F301-(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2}).*\.xml
+  file-regex: (?:.*CORESO-F301_F301-)?(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})(?:-F301-.*CORESO)?.*\.xml
   time-validity: daily
   remote-file-regex:
-    - ".*CORESO-F301_F301-(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2}).*\\.xml"
+    - "(?:.*CORESO-F301_F301-)?(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})(?:-F301-.*CORESO)?.*\\.xml"
```

Comment:
This regex works with both naming convention. The problem here is that it could accept also others files not only F301.
I suggest to keep this generic regex in the file regex but to put two different regex in the remote-file-regex
  remote-file-regex:
    - ".*CORESO-F301_F301-(?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2}).*\\.xml"
    - "(?<year1>[0-9]{4})(?<month1>[0-9]{2})(?<day1>[0-9]{2})-F301-.*CORESO.*\\.xml"
    To be tested with docker

Reply:
changed, simplified and tested ok